.ci/docker-compose-file/Makefile.local

```diff
@@ -20,17 +20,23 @@ up:
 		-f .ci/docker-compose-file/docker-compose.yaml \
 		-f .ci/docker-compose-file/docker-compose-mongo-single-tcp.yaml \
 		-f .ci/docker-compose-file/docker-compose-mysql-tcp.yaml \
+		-f .ci/docker-compose-file/docker-compose-mysql-tls.yaml \
 		-f .ci/docker-compose-file/docker-compose-pgsql-tcp.yaml \
+		-f .ci/docker-compose-file/docker-compose-pgsql-tls.yaml \
```

Comment:
Maybe this got commited accidentally? Because the PR mentions only Redis authn tests.

Reply:
Work in progress, now I purposely named the branch `authn-ssl-tests` to add backends :)
I'll update PR name and description before making it "ready for revieew" from draft.

There are three backends left:
* mongo
* http
* jwt with jwks